WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -65,9 +54,6 @@ public WalletViewModel(UiContext uiContext, IWalletModel walletModel, Wallet wal
 							.BindTo(this, x => x.IsCoinJoining)
 							.DisposeWith(Disposables);
 
-		//this.WhenAnyValue(x => x.History.IsTransactionHistoryEmpty)
-		//	.Subscribe(x => IsTransactionHistoryEmpty = x);
-
```

Comment:
What about this?

Reply:
that was a leftover from the past, I think, because it's not being used anywhere. The one that's being used is in `HistoryViewModel`

Reply:
Ohh I thought the empty history design was broken because if this is missing, but you are right, the code is there in HistoryViewModel, the only problem is it isn't working. Might be a similar bug to https://github.com/zkSNACKs/WalletWasabi/issues/11752 that an empty list doesn't have signal?
@SuperJMN 

![image](https://github.com/zkSNACKs/WalletWasabi/assets/16364053/c2fef95d-10c6-401b-8094-76e95a0f37e2)

Reply:
> Ohh I thought the empty history design was broken because if this is missing, but you are right, the code is there in HistoryViewModel, the only problem is it isn't working. Might be a similar bug to #11752 that an empty list doesn't have signal? @SuperJMN
> 
> ![image](https://user-images.githubusercontent.com/16364053/278379807-c2fef95d-10c6-401b-8094-76e95a0f37e2.png)

I'm almost sure this is the case. Let me figure out what's going on.

Reply:
It's fixed now :)